availability-store/Cargo.toml

```diff
@@ -15,15 +15,15 @@ futures = "0.3.4"
 tokio = { version = "0.2.10", features = ["rt-core"] }
 exit-future = "0.2.0"
 codec = { package = "parity-scale-codec", version = "1.1.0", features = ["derive"] }
-sc-network = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-consensus_common = { package = "sp-consensus", git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-client = { package = "sc-client-api", git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-sc-client = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-sp-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-keystore = { package = "sc-keystore", git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-master" }
+sc-network = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
Please revert.

We always point to the `polkadot-master` branch. You can try to update Substrate or just let the pr sit here until somebody else updates Substrate and pulls in your pr.

Reply:
Right, makes sense.
The PR was just not building with my substrate branch (which was already merged into master), so I was unsure how to check whether everything works out and tried using substrate `master`.
So do I switch it to `polkadot-master` and then just wait for someone to update it?

Reply:
I guess my question is: How do I verify that my PR is still valid when we merge it?

Reply:
The pr will be merged into another one, that runs CI before. So, we will not merge this one directly. If you want to have this merged, you need to update the `polkadot-master` branch in Substrate and fix all the compile errors in Polkadot.